src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/TemporaryAbsence.kt

```diff
@@ -14,8 +15,10 @@ data class TemporaryAbsence(
   val lastName: String,
   @Schema(description = "Date of birth", example = "1971-02-01")
   val dateOfBirth: LocalDate,
+  @JsonProperty("offenderNo")
   @Schema(description = "Prison number", example = "A1234AA")
   val prisonNumber: String?,
-  @Schema(description = "Reason", example = "Hospital")
+  @JsonProperty("movementReason")
```

Comment:
movementReason is a bit of a prison-api domain specific phrase - think reason is more appropriate in terms of the concept of a temporary absence.. again would probably map this across in the service layer